### ISSUE TITLE
Is it relevant to have a maximum version of DynaFlow?

### ISSUE THREAD

**Issue by rosiereflo** [powsybl/powsybl-dynawo#155]
Why is there a maximum version for DynaFlow?
I would have expected to have only a minimum version (versions of DynaFlow before this minimum version are not compatible anymore) . It means that to test the behavior with the development version of DynaFlow an user has to change powsybl code.

Independently from the answer to this question the maximum version of DynaFlow needs to be updated to v1.4.0.

**Comment by geofjamg** [comment]
You are right, we should only have a minimum version, especially if dynawo input/output formats are evolved trying to keep compatibility with the last N recent versions.

**Comment by flo-dup** [comment]
Corresponding issue created #159
